### ISSUE TITLE
Design Mode Generates Exception About not being able to find an image.

### ISSUE THREAD

**Issue by CheetahChrome** [XamlAnimatedGif/XamlAnimatedGif#74]
It can generate the below exception (VS2015 UP2: WPF .net 4.5.2 project) when it can't find an image in design time. Even setting toggling `gif:AnimationBehavior.AnimateInDesignMode="false"` has no effect on this error.

As reported by the designer:

> DirectoryNotFoundException: Could not find a part of the path 
'C:\{removed local path}\Views\Assets\LoadingGreen.gif'.

```
   at MS.Internal.AppModel.ResourcePart.GetStreamCore(FileMode mode, FileAccess access)
   at System.IO.Packaging.PackagePart.GetStream(FileMode mode, FileAccess access)
   at System.IO.Packaging.PackWebResponse.CachedResponse.GetResponseStream()
   at System.IO.Packaging.PackWebResponse.GetResponseStream()
   at System.IO.Packaging.PackWebResponse.get_ContentType()
   at System.Windows.Media.Imaging.BitmapDecoder.SetupDecoderFromUriOrStream(Uri uri, Stream stream, BitmapCacheOption cacheOption, Guid& clsId, Boolean& isOriginalWritable, Stream& uriStream, UnmanagedMemoryStream& unmanagedMemoryStream, SafeFileHandle& safeFilehandle)
   at System.Windows.Media.Imaging.BitmapDecoder.CreateFromUriOrStream(Uri baseUri, Uri uri, Stream stream, BitmapCreateOptions createOptions, BitmapCacheOption cacheOption, RequestCachePolicy uriCachePolicy, Boolean insertInDecoderCache)
   at System.Windows.Media.Imaging.BitmapImage.FinalizeCreation()
   at System.Windows.Media.Imaging.BitmapImage.EndInit()
   at XamlAnimatedGif.AnimationBehavior.CheckDesignMode(Image image, Uri sourceUri, Stream sourceStream)
   at XamlAnimatedGif.AnimationBehavior.<InitAnimationAsync>d__42.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by thomaslevesque** [comment]
Thanks for reporting this @CheetahChrome. It will be fixed ASAP.

**Comment by thomaslevesque** [comment]
This issue is fixed in version 1.1.4
